### ISSUE TITLE
Where is the sphinxext dir?

### ISSUE THREAD

**Issue by twiecki** [matplotlib/sampledoc#22]
https://matplotlib.org/sampledoc/extensions.html tries to copy the files from a sphinxext dir in this repo which doesn't seem to exist.

**Comment by timhoffm** [comment]
https://github.com/matplotlib/matplotlib/tree/master/doc/sphinxext

**Comment by mattgillucl** [comment]
Yes, but according to the tutorial, `sphinxet/` should contain the following files:
`apigen.py  docscrape.py  docscrape_sphinx.py  numpydoc.py`
And the link provided by @timhoffm doesn't have those.

**Comment by SiliconKite** [comment]
Same issue here. What a pity that the main example for sphinx is not working. Thanks anyway

**Comment by JenningsAutomation** [comment]
same issue as above. I guess nobody's watching this repository. Same issue for 3 years. No sphinxext dir

```
home:~/tmp/sampledoc> mkdir sphinxext
home:~/tmp/sampledoc> cp ../sampledoc_tut/sphinxext/*.py sphinxext/
home:~/tmp/sampledoc> ls sphinxext/
apigen.py  docscrape.py  docscrape_sphinx.py  numpydoc.py
```

**Comment by jklymak** [comment]
I agree this is largely unmaintained and is here for historical purposes.  OTOH if someone wanted to modernize it, I'm sure volunteers would be welcome.

**Comment by xiaoouwang** [comment]
For all those stuck with this problem see here https://github.com/numpy/numpydoc/tree/main/numpydoc
Based on my test the apigen.py is not necessary.
